src/pages/contact-us.js

```diff
@@ -51,21 +51,33 @@ const ContactUs = () => (
 
     <ConstrainedWidth>
       <Text size="normal" color="black" maxCharacter="49ch">
-        Follow us on Twitter
+        <p>
+          Follow us on {''}
+          <a
+            className="linkedin-link"
+            target="_blank"
+            href="https://www.linkedin.com/company/neontribe-ltd/"
+            aria-label="This link navigates to the Neontribe LinkedIn page"
+          >
+            LinkedIn.
+          </a>
+        </p>
       </Text>
 
       <div className="social-media-container">
         <div className="social-media-labels">
-          <img
-            src={Twitter}
-            height={42}
-            width={42}
-            alt="This link navigates to the @neontribe twitter page"
-          />
-
-          <Text color="#561dee" size="normal">
-            <a href="https://twitter.com/neontribe"> @neontribe</a>
-          </Text>
+          <a
+            className="linkedin-logo"
+            target="_blank"
+            href="https://www.linkedin.com/company/neontribe-ltd/"
+          >
+            <img
+              src="/icons/linkedin.png"
```

Comment:
I think this image file is missing?

Reply:
thank you! think the public file is hidden, will move my image elsewhere!